test/bats/gcp.bats

```diff
@@ -137,5 +120,7 @@ setup() {
 }
 
 teardown_file() {
+  kubectl logs -l app=csi-secrets-store-provider-gcp  --tail -1 -c provider -n kube-system > ${ARTIFACTS}/provider-gcp.logs
```

Comment:
Let's also add logs for the azure provider here? The label is `app=csi-secrets-store-provider-azure`

Reply:
created a helper and updated the azure.bats